DynamicColor/UIColor.swift

```diff
@@ -153,7 +153,7 @@ public extension UIColor {
   :returns: A lighter UIColor.
   */
   public func lightenColor(amount: CGFloat) -> UIColor {
-    var amount = min(1, max(0, amount))
+    let amount = min(1, max(0, amount))
```

Comment:
Variable( you declared) and internal parameter are same.

```swift
let _amount = 
```
It may better to understand.